connectors/cli/connector.py

```diff
@@ -116,9 +114,13 @@ async def __create_connector(
             )
             timestamp = iso_utc()
 
+            await self.__create_search_index(index_name, language)
```

Comment:
Does this mean an index is created every time? Can we still use `--from-index` and create a connector for an existing index?